### PR TITLE
Add bootstrap team meetings

### PR DESCRIPTION
**Pull request by albertlarsan68**
Add @rust-lang/bootstrap meetings

### CODE REVIEW

File: bootstrap.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = "Rust Bootstrap Team"
+description = "Meetings for the bootstrap team"
+
+[[events]]
+uid = "1705942319405"
+last_modified_on = "2024-01-22T18:00:00Z"
+title = "Rust Bootstrap Team Meeting"
+start = "2024-02-05T18:00:00Z"
```
Comment (onur-ozkan) [inline bootstrap.toml:8]: Which timezone does 18:00 refers to? 17:00 CET is equivalent to 16:00 UTC.

Comment (albertlarsan68) [reply bootstrap.toml:8]: Oops, my bad.
I did not check the output of Copilot, and it made sense in my head because timezones are confusing

Comment (onur-ozkan) [reply bootstrap.toml:8]: You can add the timezone field too.

Like here:
https://github.com/rust-lang/calendar/blob/14bb4ae00cc2cb9b152570bdb00b92ef3140c49b/infra.toml#L13

Comment (albertlarsan68) [reply bootstrap.toml]: Added new Europe/Paris TZ, and used it.

**Review by onur-ozkan — APPROVED** [review]
Thanks!

File: _timezones.toml [hunk 2]
```diff
@@ -54,3 +54,20 @@ start = "1970-03-29T02:00:00.00"
 from = "+0100"
 to = "+0200"
 rule = { frequency = "yearly", by_month = [3], by_day = [ { day = "sunday", num = -1 } ] }
+
+[[timezones]]
+name = "Europe/Paris"
```
Comment (davidtwco) [inline _timezones.toml:59]: You don't need to add this, you can just use `Europe/Amsterdam` instead, it amounts to the same timezones.

**Review by davidtwco — APPROVED** [review]

**Pull request merged**